core/src/main/scala/design/hamu/core/command/FibCommand.scala

```diff
@@ -0,0 +1,26 @@
+package design.hamu.core.command
+
+object FibCommand extends Command {
+
+	// catch parameter, convert to int, pass to fib()
+	def run(ls: List[String]): String = ls match {
+		case (head:String) :: (tail: List[String]) => {
+			fib(toInt(head)).toString
+		}
+	}
+
+	// helper function i stole off the internet
+	def toInt(s: String): Int = {
+		try {
+			s.toInt
+		} catch {
+			case e: Exception => 0
+		}
+	}
+
+	def fib(n: Int): Int = n match {
+		case 0 => 0
+		case 1 => 1
+		case _ => (fib(n-1) + fib(n-2))
```

Comment:
Btw you couldve made this tail recursion and i think it'll save some more memory